etc/sample.openstack.yaml

```diff
@@ -37,9 +37,12 @@ compute:
         defaults:
             platform: amd64
             network: public
+            extra_specs:
```

Comment:
This should be removed, as the static information and its defaults are going to be added if there is no such extra_spec in the flavor (i.e. by default all flavors would have infiniband!)